### ISSUE TITLE
ucd-theme-slim-select needs an aria-label

### ISSUE THREAD

**Issue by spelkey-ucd** [ucd-library/ucdlib-theme#107]
When passing through an attribute:
```
<ucd-theme-slim-select listbox-aria-label='Foo'></ucd-theme-slim-select>
```

That value should be passed as the aria-label property to the:
```
<div class="ss-list" role="listbox"></div>
```

that is created after SlimSelect instantiation.
